src/idd/idd.utils.js

```diff
@@ -206,14 +206,14 @@ InteractiveDataDisplay.Utils =
         
         getMinMaxForPair: function (arrayx, arrayy) {
```

Comment:
@ElenaPochernina Leave `getMinMaxForPair` as it was before but call it for `array.median`.